config/locales/en.yml

```diff
@@ -97,8 +111,8 @@ en:
   requests:
     success:
       email_notification:
-        proxy: "(We've sent a copy of this request to your email and to the designated notification address.)"
-        default: "(We've sent a copy of this request to your email.)"
+        proxy: "We'll send an email to you and to the designated notification address when processing is complete."
+        default: "We'll send you an email when processing in complete."
```

Comment:
`is complete`?  if so, appears to have been copy/pasted into some tests (which is actually how i noticed it).